README.md

```diff
@@ -65,10 +80,10 @@ APIs.
 Please submit your questions, feature requests and bug reports via
 [GitHub issues].
 
-## How to Contribute
+## How to contribute
 
 We welcome community contributions to nGraph. If you have an idea how
-to improve the Library:
+to improve it:
```

Comment:
on how to improve it
for improving it
bot sound better to me.

Reply:
Agree. Fixed